### PR TITLE
Initial code without internal info and tests

### PR DESCRIPTION
**Pull request by terrytangyuan**
This PR migrates internal code base to GitHub, excluding sensitive information, docs, examples, and tests. This PR has not been tested yet since we will need to clean up tests to remove internal data and migrate them gradually in follow-up PRs. The interface remains the same as the current version.

cc @merlintang @ywskycn @linkerzhang

Signed-off-by: terrytangyuan <terrytangyuan@gmail.com>

### CODE REVIEW

File: couler/argo.py [hunk 1]
```diff
@@ -0,0 +1,186 @@
+import atexit
```
Comment (merlintang) [inline couler/argo.py:1]: can we add the license here ?

Comment (terrytangyuan) [reply couler/argo.py:1]: See https://github.com/couler-proj/couler/pull/2 for LICENSE document. We can add license headers to each file later in a separate PR so it's clear what's changed based off internal code.

**Comment by terrytangyuan** [comment]
@merlintang Thanks. I updated the PR description accordingly.

**Pull request merged**